service/src/main/java/bio/terra/pipelines/stairway/imputation/steps/gcp/PollCromwellSubmissionStatusStep.java

```diff
@@ -33,8 +27,8 @@ public class PollCromwellSubmissionStatusStep implements Step {
   private final Logger logger = LoggerFactory.getLogger(PollCromwellSubmissionStatusStep.class);
 
   public PollCromwellSubmissionStatusStep(
-      SamService samService,
       RawlsService rawlsService,
+      SamService samService,
```

Comment:
just curious why the reorder?

Reply:
all the other steps have rawls then sam and it annoyed me this one was different 😄